Cargo.toml

```diff
@@ -46,7 +46,7 @@ serde = { version = "1.0", optional = true, features = ["derive"] }
 serde_json = { version = "1.0", optional = true }
 lazy_static = { version = "1.4", optional = true }
 once_cell = { version = "1.17", optional = true }
-mysql = { version = "24.0", optional = true }
+mysql = { version = "24.0", default_features = false, optional = true}
```

Comment:
just making sure - we don't need the `rustls-tls` right?

Reply:
we set the featureset used for mysql with a feature of our own named rustls-tls that is in the default set, so the import specifies no features at all.

Reply:
yep, just making sure